code/datums/status_effects/buffs.dm

```diff
@@ -178,7 +182,11 @@
 			itemUser.adjustBrainLoss(-1.5)
 			itemUser.adjustCloneLoss(-0.5) //Becasue apparently clone damage is the bastion of all health
 		//Heal all those around you, unbiased
+		if(heal_points < (max_heal_points))
+			heal_points = max((heal_points += 3), max_heal_points)
```

Comment:
The `+=` makes it possible to store a bit more than `max_heal_points`. For example if `heal_points` is 48, the 48 turns to 51, and then it will be `max(51, 50)`, which is 51.

Reply:
Really not sure why I did code like that, should be min, as then if heal points was 12, it goes to 15 and chooses 12, it it was 29, it goes to 52 and chooses 50. Will fix